src/window.rs

```diff
@@ -1064,8 +1064,6 @@ pub trait Window: AsAny + Send + Sync + fmt::Debug {
     ///
     /// ## Platform-specific
     ///
-    /// - **X11:** Area is not supported, only position. The bottom-right corner of the provided
```

Comment:
Need to document that in the `CHANGELOG` as well, in sort of that `offset` is no longer needed IIRC and say what the default is.

Reply:
Fair enough, the offset requirement was changed in #3966 but I agree it warrants a more significant explanation in the CHANGELOG.

Reply:
Should I remove the changelog entry from #3966 and replace it with this section?